website/_config.yml

```diff
@@ -0,0 +1,36 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+title: Carbon Language
```

Comment:
I'm guessing this is where the top-left corner "Carbon Language" comes from?

Wondering if we could say "Carbon Language Documentation" without it ruining the layout of the page... Or in some other way make it more clear that this isn't intended to be the long-term website for the project or language, but a build and render of our current documentation.

I can try getting this to render locally and play with some options if useful.

Reply:
Done.

Note, switched capitalization for https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings